src/codewhisperer/commands/onInlineAcceptance.ts

```diff
@@ -154,4 +154,7 @@ export async function onInlineAcceptance(
             )
         }
     }
+
+    // at the end of recommendation acceptance, clear recommendations.
+    RecommendationHandler.instance.clearRecommendations()
```

Comment:
Can this be placed on L63: https://github.com/aws/aws-toolkit-vscode/blob/1d781ae4c130c159f28cfb63f9cea7684edfc59b/src/codewhisperer/commands/onInlineAcceptance.ts#L63-L66 ?

That way we handle failures

Reply:
Yes